docs/tutorial/part-two/index.md

```diff
@@ -114,43 +114,43 @@ import "./src/styles/global.css"
 // require('./src/styles/global.css')
 ```
 
-> Note: Both CommonJS (`require`) and ES Module (`import`) syntax work here. If you’re not sure which to choose, we use `import` most of the time.
+> Uwaga: Zarówno syntax CommonJS (`require`) jak Moduł ES (`import`), zadziałają w tym wypadku. Jeśli nie jesteś pewien który z nich wybrać, my w większości wypadków używamy 'import'.
 
-3. Start the development server:
+3. Uruchom serwer deweloperski (development server):
```

Comment:
[Dla spójności:](https://github.com/gatsbyjs/gatsby-pl/pull/6/files#diff-d20f31bd083ac5610b5bf5df061ad2dcR71)
```suggestion
3. Uruchom serwer developerski (development server):
```

Reply:
Ja myślę, że powinniśmy zostać przy pisowni przez 'w'. Jak po Polsku to po Polsku :) co sądzicie?

Reply:
Mi także bardziej pasuje pisownia przez `w`

Reply:
Zaraz, ale zostawiłeś sugestię zmiany na 'v'? Czy czegoś nie rozumiem ?:)

Reply:
```suggestion
3. Uruchom Serwer Deweloperski (development server):
```

Reply:
@maciekgrzybek zostawiłem sugestię, bo gdzie indziej używamy pisowni przez `v`, więc dla spójności powinniśmy się trzymać jednej pisowni.

Nie zmienia to faktu, że osobiście preferuję pisownię przez `w` - proponuję razem zdecydować i dokonać zmianę w innych częściach naszych tłumaczeń. 

Usunę tę powyższą sugestię, bo wydaje mi się że chyba [wolimy pisownię](https://github.com/gatsbyjs/gatsby-pl/pull/8/files#diff-e4405b2d7facba43db450e0b7089654fR96) przez `w` .

Reply:
Aaa spoko rozumiem :) wszystko jasne :)

Reply:
@matborowiak może lepiej nie edytujmy komentarzy, tylko dodawajmy nowe? będziemy mieli jaśniejjszą sytuację i historie komentarzy krok po kroku.

Reply:
@maciekgrzybek jestem za. Chyba że literówki czy sugestje zmian w przypadku jakieś oczywistej wpadki :)